src/test/java/SeleniumTests.java

```diff
@@ -0,0 +1,151 @@
+import org.openqa.selenium.support.ui.Select;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class SeleniumTests {
+
+    private String password = "Tester123456!";
+    private String homePage = "https://passport.yandex.com/";
+
+    private final List<Employee> employeeList = new ArrayList<>();
+
+    WebDriver driver;
+
+
+    @BeforeAll
+    static void setChromeDriverExecutable() {
+        WebDriverManager.getInstance(ChromeDriver.class).setup();
+    }
+
+    @BeforeEach
+    void setUp() {
+
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"selenium.t", "selenium.t2"})
+    void goToLoginPageAndLogIn(String testName) throws InterruptedException {
+        driver.get(homePage);
+        driver.findElement(By.id("passp-field-login")).sendKeys(testName);
+        driver.findElement(By.id("passp:sign-in")).click();
+        Thread.sleep(1000); //Thread.sleep is an implicit waiter
+        driver.findElement(By.id("passp-field-passwd")).sendKeys(password);
+        driver.findElement(By.id("passp:sign-in")).click();
+        driver.findElement(By.xpath("//*[@class='UserID-Avatar ']")).click();
+        FluentWait<WebDriver> wait = new FluentWait<>(driver).withTimeout(Duration.ofSeconds(10)).pollingEvery(Duration.ofSeconds(2)).ignoring(NoSuchElementException.class);
+        WebElement nameElement = wait.until(ExpectedConditions.visibilityOfElementLocated((By.xpath("//*[contains (@class, 'Subname')]")))); //Tried multiple identifiers, could not manage to target element
+        assertTrue(nameElement.isDisplayed());
+    }
+
+    @Test
+    void multiSelectDemoTest() {
+        driver.get("https://demo.seleniumeasy.com/basic-select-dropdown-demo.html");
+        WebElement dropDownElement = driver.findElement(By.id("multi-select"));
+        Select selector = new Select(dropDownElement);
+        List<String> listOfOptions = new ArrayList<>();
+        for (WebElement item : selector.getOptions())
+            listOfOptions.add(item.getText());
+
+        List<String> expectedSelectedOptionsList = new ArrayList<>();
+        for (int i = 0; i < 3; i++) {
+            int randomNumber = ThreadLocalRandom.current().nextInt(1, listOfOptions.size() - 1);
+            expectedSelectedOptionsList.add(listOfOptions.get(randomNumber));
+        }
+        for (String item : expectedSelectedOptionsList) {
+            selector.selectByValue(item);
+        }
+        List<String> actualSelectedOptionsList = new ArrayList<>();
+        for (WebElement selectedItem : selector.getAllSelectedOptions())
+            actualSelectedOptionsList.add(selectedItem.getText());
+
+        assertTrue(expectedSelectedOptionsList
```

Comment:
using stream api your test may look shorter
```
@Test
    public void multiSelectTest() {
        driver.get("https://demo.seleniumeasy.com/basic-select-dropdown-demo.html");
        Select multiSelect = new Select(driver.findElement(MULTI_SELECT));
        List<String> expectedStates = new Random().ints(3, 0, multiSelect.getOptions().size()).boxed()
                .map(i -> multiSelect.getOptions().get(i).getText()).collect(Collectors.toList());
        expectedStates.forEach(multiSelect::selectByValue);
        List<String> selectedStates = multiSelect.getAllSelectedOptions().stream().map(WebElement::getText)
                .collect(Collectors.toList());
        assertTrue(expectedStates.containsAll(selectedStates));
    }
```
look through and try to figure out what's going on here